### DISCUSSION TITLE
How to avoid the truncation of the Git console (with long output) when creating a commit?

### DISCUSSION THREAD

**Discussion by jandresfg** [NeogitOrg/neogit#1516] [Q&A] [answered]
Hi! I'm working on a codebase that uses pre-commit hooks, and the Git console output gets pretty long, so it switches to just displaying `[Output too long - Truncated]`, as shown in the video below. This keeps me from seeing if the hooks passed or failed. Is there a way to avoid this? Thanks! I loooove this plugin and I'd rather stay here than opening a terminal and committing from there.

https://github.com/user-attachments/assets/d14efab8-dbbe-4015-beeb-3590d28882f2

**Comment by CKolkey** [comment, marked as answer]
Heh, update to latest `master` - I juuuuuuust fixed that a day or two ago.

**Reply by jandresfg** [reply]
it worked! thank you so much! Love from 🇨🇴

**Reply by CKolkey** [reply]
Very very welcome 🥰
